src/resdk/resources/background_task.py

```diff
@@ -0,0 +1,73 @@
+"""Process resource."""
+import logging
+from time import sleep, time
+
+from resdk.exceptions import ResolweServerError
+
+from .base import BaseResource
+
+
+class BackgroundTask(BaseResource):
+    """Background task resource.
+
+    :param resolwe: Resolwe instance
+    :type resolwe: Resolwe object
+    :param model_data: Resource model data
+
+    """
+
+    endpoint = "task"
+
+    READ_ONLY_FIELDS = BaseResource.READ_ONLY_FIELDS + (
+        "pk",
+        "started",
+        "finished",
+        "status",
+        "description",
+        "output",
+    )
+    WRITABLE_FIELDS = ()
+
+    def __init__(self, resolwe, **model_data):
+        """Initialize attributes."""
+        self.logger = logging.getLogger(__name__)
+
+        #: started
+        self.started = None
+        #: finished
+        self.finished = None
+        #: status - Possible values:
+        #: WA (waiting)
+        #: PR (processing)
+        #: OK (done)
+        #: ER (error)
+        self.status = None
+        #: description
+        self.description = None
+        #: output
```

Comment:
Would be nice to know / document what the output is. A string, number, json with complex structure, etc...?

Reply:
I have added a short comment. It is JSON but the actual value depends on the background task. Currently, only the duplicate task is supported and for it, the value can be:
- a list of ids of duplicated objects on success
- a string with error details on failure

Some other background tasks (in the future) may use output differently.